transports/sql/sql-native.md

```diff
@@ -5,17 +5,17 @@ reviewed: 2018-04-23
 component: SqlTransportNative
 ---
 
-Provides low-level access to the [SQL Server Transport](/transports/sql/) with no NServiceBus or SQL Transport reference required.
+SQL Server Transport Native is a shim providing low-level access to the [SQL Server Transport](/transports/sql/) with no NServiceBus or SQL Server Transport reference required. 
 
 
 ## Usage scenarios
 
- * **Error or Audit queue handling**: Since NServiceBus requires the message type it is not suitable for use when processing an error or audit queue. SQL Native allows manipulation or consumption of messages without the need of message types.
- * **Corrupted or malformed messages**: This project operates as close to the SQL database as possible. As such message headers and body are treated as a raw string and byte array. This means corrupted or malformed messages can be read and manipulated so as to correct any problems.
- * **Deployment or decommission**: Running [installers](/nservicebus/operations/installers.md) requires starting a full endpoint. This is not always ideal during the execution of a deployment or decommission. SQL Native allows creating or deleting of queues with no running endpoint, and with significantly less code. This also makes it a better candidate for usage in deployment scripting languages like PowerShell.
+ * **Error or Audit queue handling**: Allows to easily consume messages from error and audit queues, for example to move them to a long-term archive. Since NServiceBus expects to have a queue per message type it is not suitable for use when processing an error or audit queues, which contain messages of various types. SQL Native allows manipulation or consumption of queues with multiple types of messages.
```

Comment:
> Since NServiceBus expects to have a queue per message type

this makes no sense.

It is actually about not needing the message types at runtime.